frontend/components/notifications/management/NotificationsManagementDashboards.vue

```diff
@@ -332,22 +356,25 @@ const handleDelete = (payload: Parameters<typeof deleteDashboardNotifications>[0
         <template #expansion="slotProps">
           <div class="expansion">
             <div class="info">
-              <div class="label">
+              <div :class="['label', slotProps.data.is_archived ? 'text-disabled' : '']">
```

Comment:
more concise and more explicit which parts are conditional which are not
```suggestion
              <div 
              class="label"
              :class="{'text-disabled' : slotProps.data.is_archived }">
```

Reply:
Done